hyperspyui/mainwindowbase.py

```diff
@@ -376,6 +371,17 @@ def select_tool(self, tool):
         self.active_tool = tool
         tool.connect_windows(self.figures)
 
+    def setUpdatesEnabled(self, value, force_setting=False):
+        """
+        pyqt5 doesn't seem to suffer from screen flickering and it is not 
```

Comment:
This comment is not very helpful to those who don't know what type of flickering is mentioned. Is this really needed? What problem does it solve? Also, this text is probably better as a comment than a docstring?